Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "tui"
-version = "0.8.0"
+version = "0.8.1"
```

Comment:
The version should probably not be bumped here.

Reply:
Can I somehow drop that commit from the PR or do I open another one?

Reply:
I would just drop that commit from the PR. If you need help with that then let me know :)

Reply:
I did a force push to my fork now, is there a better alternative? And this is probably off-topic, but could you explain to me, why the version bump is not necessary?

Reply:
Yeah, that's the way I would do it.

There's no reason to bump the version number since that's more of the maintainers job and it's easy for them to do. Plus, you wouldn't want to create conflicts with other pull requests that do the same.

Reply:
Well, that makes sense. Thank You!